api/src/db/migrations/v82_add_up_and_running_owning_operating_phase.test.ts

```diff
@@ -0,0 +1,46 @@
+import {
+  generatev81FormationFormData,
+  generatev81ProfileData,
+  generatev81User,
+  v81ProfileData,
+  v81UserData,
+} from "./v81_add_completed_filing_payment";
+import { migrate_v81_to_v82 } from "./v82_add_up_and_running_owning_operating_phase";
+
+describe("migrate_v81_to_v82", () => {
+  it("updates operating phase to up_and_running_owning from up_and_running for owning businessPersona", () => {
+    const v81 = makeUserData({ businessPersona: "OWNING", operatingPhase: "UP_AND_RUNNING" });
+    const v82 = migrate_v81_to_v82(v81);
+    expect(v82.profileData.operatingPhase).toEqual("UP_AND_RUNNING_OWNING");
+  });
```

Comment:
maybe not strictly necessary, but it feels nice to me to cover the additional cases:
- when OWNING but not up-and-running, operating phase should remain the same
- when UP_AND_RUNNING and starting, operating phase should remain the same